js/scripts.js

```diff
@@ -0,0 +1,15 @@
+var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.";
+var animal = "Zielone słonie";
+var animalUpperCased = animal.toUpperCase();
+
+console.log(animalUpperCased);
+
+var firstWordfromText = text.slice(0,6);
+var textCharsAfter = text.replace(firstWordfromText, animalUpperCased);
+
+var halfOfText = textCharsAfter.length/2;
+console.log("Długość połowy tekstu: " + halfOfText);
+
+var newText = textCharsAfter.substr(0, 45); // możemy użyć również slice()
```

Comment:
To samo tutaj z `45` - znajdź sposób aby określić połowę długości tekstu, a nie wbijać `45` z palca.